olsrd/files/olsrd6.init

```diff
@@ -10,6 +10,17 @@ OLSRD=olsrd6
 CONF=/var/etc/$OLSRD.conf
 PID=/var/run/olsrd6.pid
 
+wait_for_network()
```

Comment:
The wait_for_network() function is identical to the one in olsrd4.init. Think it should be put into lib/functions/olsrd.sh

same might be usefull for the body of boot() too

Reply:
I'm unsure, I will merge this fix first, since it was approved and then tidy up again.

Reply:
feel free to open a pr